docs/learn/learn-staking.md

```diff
@@ -1,324 +1,390 @@
+be held accountable. In case of misbehavior, participants in the staking process can be punished or
+**slashed**, and depending on the gravity of the situation, their stake can be partly or fully
+confiscated by the network. It is not in a staker's economic interest to orchestrate an attack and
+risk losing tokens. Any rational actor staking on the network would want to get rewarded, and the
+PoS network rewards good behavior and punishes bad behavior.
+
+## Nominated Proof-of-Stake (NPoS)
+
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} implements
+[Nominated Proof-of-Stake (NPoS)](learn-consensus.md/#nominated-proof-of-stake), a relatively novel
+and sophisticated mechanism to select the validators who are allowed to participate in its
+[consensus](learn-consensus.md) protocol. The NPoS encourages
+{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} holders to participate as nominators. NPoS
+encourages {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} holders to participate as
+**nominators**.
 
 Any potential validators can indicate their intention to be a validator candidate. Their candidacies
-are made public to all nominators, and a nominator in turn submits a list of up to
+are made public to all nominators, and a nominator, in turn, submits a list of up to
 {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominations" defaultValue={16}/> :polkadot }}
 {{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }}
-candidates that it supports. In the next era, a certain number of validators having the most
-{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} backing get elected and become active. As a
-nominator, a minimum of
+candidates that it supports, and the network will automatically distribute the stake among
+validators in an even manner so that the economic security is maximized. In the next era, a certain
+number of validators having the most {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} backing
+get elected and become active. For more information about the election algorithm go to
+[this](learn-phragmen.md) page on the wiki or
+[this](https://research.web3.foundation/en/latest/polkadot/NPoS/1.%20Overview.html?highlight=proportional%20justified%20representation#)
+research article. As a nominator, a minimum of
 {{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}
 {{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }}
-is required to submit an intention to nominate. The nomination intents are placed in a semi-sorted
-list called [bags-list](https://github.com/paritytech/substrate/pull/9507).
+is required to submit an intention to nominate. Note that in NPoS the stake of both nominators and
```

Comment:
...is required to submit an intention to nominate, **which can be thought of as registering to be a nominator.**

(trying to make clear what "intention" is.

Reply:
Specified